Magical-Kitties/Magical-Kitties-Save-the-Day.css

```diff
@@ -162,196 +169,178 @@ h1, h2, h3, h4, h5, h6, .redtext, .pinktext, .whiteonredtext {
+  grid-column: 1 / 13;
 }
 
-.fortywide {
-  display: inline-block;
-  max-width: 37%;
-  width: 37%;
+.roll-talent-powers {
+  grid-column: 5 / 13;
 }
 
-.halfwide {
-  display: inline-block;
-  max-width: 46%;
-  width: 46%;
-}
+/* Miscellaneous formatting */
 
-.threequarterswide {
-  display: inline-block;
-  max-width: 80%;
-  width: 80%;
+div.trackables {
+  display: grid;
+  grid-column: 1 / 8;
+  gap: inherit;
+  margin: 0;
+  padding: 0;
 }
 
-.fullwide {
-  display: inline-block;
-  max-width: 96%;
-  width: 96%;
+.trackables-list {
+  grid-column: 1 / 3;
 }
 
-
-/* Miscellaneous formatting */
-
-.inline-block {
-  display: inline-block;
+.trackables-limits {
+  grid-column: 3;
 }
 
 .no-display {
   display: none;
 }
 
-input[type="text"].input-large{
-   font-size: 24pt; 
-   justify-content: left;
-   align-items: left;
-   text-align: left;
+input[type="text"].input-large {
+  font-size: 24pt;
+  justify-content: left;
+  align-items: left;
+  text-align: left;
 }
 
-input[type="number"].input-large {
-   font-size: 24pt; 
-   justify-content: right;
-   align-items: right;
-   text-align: right;
+.trackables input[type="number"].input-large {
+  font-size: 24pt;
+  text-align: right;
+  height: 5rem;
+  width: 8rem;
+  margin-left: auto;
 }
 
 input[type="text"].input-medium {
-   font-size: 18pt; 
-   justify-content: left;
-   align-items: left;
-   text-align: left;
+  font-size: 18pt;
+  justify-content: left;
+  align-items: left;
+  text-align: left;
 }
 
 input[type="number"].input-medium {
-   font-size: 18pt; 
-   justify-content: right;
-   align-items: right;
-   text-align: right;
-}
-
-input[type="number"].46h{
-  height: 46.5px;
+  font-size: 18pt;
   justify-content: right;
   align-items: right;
   text-align: right;
 }
 
-textarea.kitty-desc{
+.trackables input[type="number"].limit {
+  margin-left: unset;
+  margin-right: auto;
+}
+
+textarea.kitty-desc {
   min-height: 260px;
   font-size: 18pt;
-  margin: 5px 0px;
+  margin: 5px 0 0 0;
+  width: 96.5%;
+}
+
+textarea.fullwide {
+  width: 98.5%;
 }
 
-.repcontainer textarea{
-  line-height: 20px;
+div.repcontainer {
+  grid-column: 1 / 13;
+  padding-left: 0;
+}
+
+.repcontainer textarea {
+  line-height: 32px;
   font-size: 18pt;
   height: 120px;
+  width: 99%;
+  grid-column: 1 / 13;
+}
+
+.repitem {
+  display: grid;
+  grid-template-columns: repeat(12, 1fr);
+  width: 100%;
+}
+
+.repcontrol {
+  grid-column: 1 / 13;
 }
 
 hr {
-    display: block;
-    height: 10px;
-    border: 0;
-    border-top: 10px solid #ccc;
-    margin: 1em 0;
-    padding: 0;
+  grid-column: 1 / 13;
+  height: 7px;
+  border: none;
+  margin: 20px;
+  background-image: linear-gradient(to right, #FFC5BB, #FEDCC1, #F5F4C1, #C1FFC1, #E0FFFF, #DEDEFF, #FFBBFF);
 }
 
+body.sheet-darkmode input,
+body.sheet-darkmode optgroup,
+body.sheet-darkmode select,
+body.sheet-darkmode textarea,
+body.sheet-darkmode .charsheet {
+  color: #EEE;
+  background: #1F1F1F;
+}
```

Comment:
I originally included the roll buttons in the dark mode styling, but they were losing the linear-gradient and the hover color shift looked bad.  Since they're small elements, I think it's fine to leave them to be the same as the light-mode styling.